ulib/fs/VS/Makefile

```diff
@@ -29,9 +28,7 @@ all: install-packages
 	$(MSBUILD) UlibFS.sln
 
 install-packages:
-	mkdir -p packages
-	$(NUGET) restore UlibFS.sln
-	find packages -name '*.exe' -exec chmod +x '{}' ';'
+	$(MSBUILD) /t:Restore UlibFS.sln
```

Comment:
Why do we need this change? Is `msbuild /t:Restore` equivalent in that it runs `nuget restore`, and sets correct permissions on the `.exe` files?

Reply:
> `msbuild /t:Restore` equivalent in that it runs `nuget restore`. 

That's true.

> and sets correct permissions on the .exe files? 

That's me not finding what kind of "EXE" is affected by this line. I suspect that this is line maybe related to how old Mono was working. I cannot find any package which have EXE inside.